### PR TITLE
Add new page descriptions

### LINKED ISSUE #8812: Create new page descriptions
Write page descriptions for all pages on polaris.shopify.com that are currently missing them. Use this [audit](https://docs.google.com/spreadsheets/d/1HuTfSqs_OqNe5AotrJpZj8TbRuRb6PRLnaUmGa8TOk8/edit?usp=sharing) as a reference.

### PR DESCRIPTION
**Pull request by jjgali**
### WHY are these changes introduced?

Fixes #8812

### WHAT is this pull request doing?

Adds missing page descriptions to pages in polaris.shopify.com.

### CODE REVIEW

File: polaris.shopify.com/content/components/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Components
+description: Components are the reusable building blocks for creating Shopify admin experiences.
```
Comment (jjgali) [inline polaris.shopify.com/content/components/index.md:3]: For some reason I'm not seeing this one when I view the hosted site.

File: polaris.shopify.com/content/icons.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Icons
+description: Use icons to increase visual appeal and improve navigation, comprehension, and engagement.
```
Comment (jjgali) [inline polaris.shopify.com/content/icons.md:3]: Not seeing this description on the hosted site.

File: polaris.shopify.com/content/tokens.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Tokens
+description: Tokens are variables that represent design decisions such as color, typography, and spacing, in a consistent and reusable way.
```
Comment (jjgali) [inline polaris.shopify.com/content/tokens.md:3]: Not seeing this description on the hosted site.

**Comment by jjgali** [comment]
Would love some 👁️'s on this! I'm trying to add descriptions to all pages on polaris.shopify.com, but a few aren't showing up. Specifically, Components, Icons, and Tokens aren't showing descriptions on the hosted site. Are there additional steps needed for that to happen?

**Review by martenbjork — APPROVED** [review]
💯

**Comment by martenbjork** [comment]
Fantastic! I think the reason some of the descriptions aren't showing up is simply because the website in very inconsistent in how it uses the `description` data. Some templates use it, some done.

The updated version of the site _always_ uses the descriptions, so they will automatically show up on there :)

**Pull request merged**